build/Containerfile

```diff
@@ -5,11 +5,25 @@ RUN echo -e "[almalinux9-appstream]" \
  "\nbaseurl = https://repo.almalinux.org/almalinux/9/AppStream/\$basearch/os/" \
  "\nenabled = 1" \
  "\ngpgcheck = 0" > /etc/yum.repos.d/almalinux.repo
+
 RUN dnf -y install postgresql && dnf clean all
 USER 1001
-COPY --chown=1001:1001 . /kai
```

Comment:
@JonahSussman did you run into an issue that prompted the change to break out what is explicitly copied?

Reply:
Thank you, makes sense.